src/Sarif.Driver/Sdk/AnalyzeOptionsBase.cs

```diff
@@ -21,11 +21,6 @@ public abstract class AnalyzeOptionsBase : CommonOptionsBase
             HelpText = "File path to which analysis output will be written.")]
         public string OutputFilePath { get; set; }
 
-        [Option(
-            "verbose",
-            HelpText = "Emit verbose output. The resulting comprehensive report is designed to provide appropriate evidence for compliance scenarios.")]
-        public bool Verbose { get; set; }
```

Comment:
>public bool Verbose { get; set; } [](start = 8, length = 33)

since we are removing and that is a breaking change, in src/ReleaseHistory, add a new line, pls

Reply:
Done.

---
In reply to: [559049254](https://github.com/microsoft/sarif-sdk/pull/2246#discussion_r559049254) [](ancestors = 559049254)